Source/WebCore/rendering/RenderLayerBacking.cpp

```diff
@@ -779,7 +779,7 @@ void RenderLayerBacking::updateBackdropFiltersGeometry()
 
     FloatRoundedRect backdropFiltersRect;
     if (renderBox->style().hasBorderRadius() && !renderBox->hasClip()) {
-        auto roundedBoxRect = renderBox->roundedBorderBoxRect();
+        auto roundedBoxRect = renderBox->borderRoundedRect();
```

Comment:
The naming of these is very confusing, but I think this makes sense :)

Could the test be a WPT test instead of a WebKit internal test?

Reply:
I will try to add WPT test case.